geth/utils/validation.py

```diff
@@ -145,3 +145,35 @@ def validate_genesis_data(genesis_data: GenesisDataTypedDict) -> bool:
     except TypeError as e:
         raise PyGethValueError(f"error while validating genesis_data: {e}")
     return True
+
+
+def fill_default_genesis_data(
+    genesis_data: GenesisDataTypedDict,
+) -> GenesisData:
+    """
+    Fills in default values for the genesis data
+    """
+    if genesis_data.get("config"):
+        try:
+            genesis_data_config_filled = GenesisDataConfig(**genesis_data["config"])
+        except ValidationError as e:
```

Comment:
Do you think it's worth adding a test for hitting the error conditions?

Reply:
yes! added.